src/main/java/hudson/plugins/promoted_builds/conditions/ManualCondition.java

```diff
@@ -96,17 +99,31 @@ public ParameterDefinition getParameterDefinition(String name) {
     }
 
     public Set<String> getUsersAsSet() {
-        if (users == null || users.equals(""))
+        return getUsersAsSet(null);
+    }
+
+    public Set<String> getUsersAsSet(AbstractBuild<?,?> build) {
+        if (users == null || users.equals("") || build == null)
```

Comment:
This condition is not correct from what I see.
When `build` is null, the result will be always empty.
E.g. `getUsersAsSet() ` will be always returning empty list after the change

Reply:
What sensitive environment variable is exposing right now?
Could you give me some examples?